src/Bielu.Examine.Umbraco/ElasticsearchExamineIndexRebuilder.cs

```diff
@@ -164,10 +168,16 @@ private void RebuildIndex(string indexName, TimeSpan delay, CancellationToken ca
 
                     populator.Populate(index);
                 }
+
                 if (index is IBieluExamineIndex elasticIndex)
                 {
+                    // Reset the ExamineManagementController cache for 1 second  
+                    var cacheKey = "temp_indexing_op_" + indexName;
+                    _runtimeCache.Insert(cacheKey, () => "tempValue", TimeSpan.FromSeconds(1));
```

Comment:
Can you explain to me why we doing it?

Reply:
The DocumentCount = 0  when the metadata is retrieved by the ExamineManagementController. Adding 1 second delay prevents this.

Reply:
that wouldn't add delay of 1s, it just add key to do memory cache which takes max 100ms 🤔

Reply:
oh i see what you doing here, you forcing umbraco to not check indexes for 1, not sure that is correct approach.

Reply:
Maybe not, but it works great in my case, the metadata is correct every time I reindex so far. If you have a better solution, I'd love to hear about it.